cmd/package-image/main.go

```diff
@@ -128,6 +154,11 @@ func main() {
 		logger.Errorf(err.Error())
 		os.Exit(1)
 	}
+	err = (&p).runSteps(processExtraTags())
```

Comment:
This is so that if skipIfImageArtifactExists short circuits the first runSteps loop extra tags are still processed if their artifact are not yet set.